modules/nf-core/sourmash/taxannotate/meta.yml

```
@@ -0,0 +1,59 @@
+name: "sourmash_taxannotate"
+description: Annotate list of metagenome members (based on sourmash signature matches) with taxonomic information.
+keywords:
+  - FracMinHash sketch
+  - signature
+  - kmer
+  - containment
+  - sourmash
+  - genomics
+  - metagenomics
+  - taxonomic classification
+  - taxonomic profiling
+tools:
+  - "sourmash":
+      description: Compute and compare FracMinHash signatures for DNA data sets.
+      homepage: https://sourmash.readthedocs.io/
+      documentation: https://sourmash.readthedocs.io/
+      tool_dev_url: https://github.com/sourmash-bio/sourmash
+      doi: "10.21105/joss.00027"
+      licence: ["BSD-3-clause"]
+
+## Description of all of the variables used as input
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - gather_results:
+      type: file
+      description: |
+        Table with signatures classified as belonging to any of the genomes
+        in the sourmash database(s), result of `sourmash gather` command.
+  - taxonomy:
+      type: file
+      description: Database with lineages (in CSV format)
```

Although it's not clear from the docs, the `--taxonomy` flag can take multiple taxonomy csv files, so maybe something like:
```suggestion
      description: One or more databases with lineages (in CSV format)
```